web/pages/create.tsx

```diff
@@ -129,6 +133,18 @@ export function NewContract(props: {
 
   const min = minString ? parseFloat(minString) : undefined
   const max = maxString ? parseFloat(maxString) : undefined
+  const initialValue = initialValueString
+    ? parseFloat(initialValueString)
+    : undefined
+
+  const adjustIsLog = () => {
+    if (min === undefined || max === undefined) return
+    const lengthDiff = Math.abs(Math.log10(max) - Math.log10(min))
```

Comment:
You should use `log10(max - min)` instead. It recommends everything as log scale if min = 0